# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/StreamLoader.cpp
# Commit: 313c1f782833
# Full Hash: 313c1f7828336b6163271454aaface96f0e3fc96
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-10-02 09:50:09
# Regressor Bug: 1899734
# File Overlap Count: 1
# Description:
#   Bug 1899734 - Part 5: Make SharedSubResourceCache capable of handling CacheablePerformanceTimingData. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223351
# ==============================================================================

diff -r baa464d5f255 -r 313c1f782833 layout/style/StreamLoader.cpp
--- a/layout/style/StreamLoader.cpp	Tue Oct 01 12:17:21 2024 +0000
+++ b/layout/style/StreamLoader.cpp	Tue Oct 01 12:17:22 2024 +0000
@@ -113,6 +113,9 @@
   // Resolution of parse promise fires onLoadEvent and this should not happen
   // before main thread OnStopRequest is dispatched.
   if (NS_IsMainThread()) {
+    mSheetLoadData->mNetworkMetadata =
+        new SubResourceNetworkMetadataHolder(aRequest);
+
     if (mOnDataFinishedTime) {
       // collect telemetry for the delta between OnDataFinished and
       // OnStopRequest